### PR TITLE
AMLS-4914

### PR DESCRIPTION
**Pull request by owen-opencast**
Updated components in Business Details flow, except for pages that need splitting / more work:

http://localhost:9222/anti-money-laundering/business-details/business-address
http://localhost:9222/anti-money-laundering/business-details/business-email-address
http://localhost:9222/anti-money-laundering/business-details/address-for-letters

## How Has This Been Tested?

TBD

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/businessdetails/contacting_you_phone.scala.html [hunk 1]
```diff
@@ -24,24 +24,28 @@
...
 ) {
 
-    <p>@Messages("businessdetails.contactingyou.phone.lbl2")</p>
-
     @form(f, controllers.businessdetails.routes.ContactingYouPhoneController.post(edit)) {
 
-        @fieldset(
+
+        @fieldsetWithErrorGroup(
             f = f("contactingyou-fieldset"),
```
Comment (drewmiley) [inline app/views/businessdetails/contacting_you_phone.scala.html]: Remove -fieldset

File: app/views/businessdetails/contacting_you_phone.scala.html [hunk 2]
```diff
@@ -24,24 +24,28 @@
...
-            legend = "businessdetails.contactingyou.phone.title",
-            legendHidden = true,
+            heading = "businessdetails.contactingyou.phone.title",
+            section = "summary.businessdetails",
+            supportingContent = supportingContent,
             panel = false
         ) {
+
```
Comment (drewmiley) [inline app/views/businessdetails/contacting_you_phone.scala.html]: Remove lines

File: app/views/businessdetails/letters_address.scala.html [hunk 3]
```diff
@@ -25,42 +25,37 @@
...
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
-                )
-            }
+            @radio(
+                f = f("lettersAddress"),
+                labelText = "lbl.yes",
+                value = "true",
+                inline = true
```
Comment (drewmiley) [inline app/views/businessdetails/letters_address.scala.html]: Remove inline on radio buttons

File: app/views/businessdetails/previously_registered.scala.html [hunk 4]
```diff
@@ -36,44 +35,38 @@
...
         ) {
-
-            @errorGroup(
-                field = f("previouslyRegistered")
+            @radio(
+                f = f("previouslyRegistered"),
+                labelText = "lbl.yes",
+                value = "true",
+                target = s"#$target",
+                inline = true)
```
Comment (drewmiley) [inline app/views/businessdetails/previously_registered.scala.html]: Remove inline

File: app/views/businessdetails/vat_registered.scala.html [hunk 5]
```diff
@@ -36,44 +35,37 @@
 
     @form(f, controllers.businessdetails.routes.VATRegisteredController.post(edit)) {
 
-        @fieldset(
+        @fieldsetWithErrorGroup(
             f = f("registeredforvat-fieldset"),
```
Comment (drewmiley) [inline app/views/businessdetails/vat_registered.scala.html]: Change to registeredforVAT

File: app/views/businessdetails/contacting_you_phone.scala.html (hunk 1 above)
Comment (owen-opencast) [reply app/views/businessdetails/contacting_you_phone.scala.html]: Done

File: app/views/businessdetails/contacting_you_phone.scala.html (hunk 2 above)
Comment (owen-opencast) [reply app/views/businessdetails/contacting_you_phone.scala.html]: Done

File: app/views/businessdetails/letters_address.scala.html (hunk 3 above)
Comment (owen-opencast) [reply app/views/businessdetails/letters_address.scala.html]: Done

File: app/views/businessdetails/previously_registered.scala.html (hunk 4 above)
Comment (owen-opencast) [reply app/views/businessdetails/previously_registered.scala.html]: Done

File: app/views/businessdetails/vat_registered.scala.html (hunk 5 above)
Comment (owen-opencast) [reply app/views/businessdetails/vat_registered.scala.html]: Done

File: app/views/businessdetails/contacting_you_phone.scala.html [hunk 6]
```diff
@@ -24,22 +24,22 @@
...
 
-    <p>@Messages("businessdetails.contactingyou.phone.lbl2")</p>
-
     @form(f, controllers.businessdetails.routes.ContactingYouPhoneController.post(edit)) {
 
-        @fieldset(
-            f = f("contactingyou-fieldset"),
-            legend = "businessdetails.contactingyou.phone.title",
-            legendHidden = true,
+        @fieldsetWithErrorGroup(
```
Comment (drewmiley) [inline app/views/businessdetails/contacting_you_phone.scala.html]: Apologies I missed this. The fieldsetWithErrorGroup is not required and we only need the input component. See here for a similar implementation https://github.com/hmrc/amls-frontend/blob/master/app/views/businessactivities/business_employees.scala.html

Comment (owen-opencast) [reply app/views/businessdetails/contacting_you_phone.scala.html]: Done

**Pull request merged**